content/24.vaccines-app.md

```diff
@@ -392,31 +393,31 @@ Reactogenicity was mostly absent and of short duration.
 The two-dose vaccine regimen induced anti-spike IgG levels and neutralizing antibody-titers exceeding those in the convalescent plasma of symptomatic patients.
 In line with the preclinical studies, the use of Matrix-M adjuvant further increased anti-spike immunoglobulin levels and induced a Th1 response.
 The outcomes of this trial suggest that Novavax-CoV2373 has an acceptable safety profile and is able to induce a strong immune response with high neutralizing antibody titers.
-The phase II component of this phase I/II trial was recently uploaded to an open-access repository [@doi:10.1101/2021.02.26.21252482].
-This part of the trial was designed to identify which dosing regimen should move forward into late phase clinical trials.
+The phase II component of the trial was designed to identify the dose regimen for the next clinical trial stage [@doi:10.1371/journal.pmed.1003769].
 Both younger (18-59 years) and older patients (60-84 years) were randomly assigned to receive either 5 &mu;g or 25 &mu;g Novavax-CoV2373 or placebo in two doses, 21 days apart.<!--To Do: What is placebo?-->
 In line with the phase I data, reactogenicity remained mild to moderate and of short duration.
 Both dose levels were able to induce high anti-spike IgG titers as well as neutralizing antibody responses after the second dose.
 Based on both safety and efficacy data, the 5 &mu;g dosing regimen was selected as the optimal dose regiment for the ongoing phase III trial.
-Although the phase III trial data has not been published yet, Novavax announced an efficacy of 89.3% based on their phase 3 trial in the UK and South Africa.
+Although the phase III trial data has not been published yet, Novavax announced an efficacy of 89.3% based on their phase III trial in the UK and South Africa.
 This trial included over 15,000 participants in the UK and 4,000 participants in South Africa with occurrence of a PCR-confirmed symptomatic case as the primary endpoint.
 In the first interim analysis (U.K.), 56 cases of COVID-19 were observed in the placebo group compared to 6 cases in the treatment group.
 Importantly, the vaccine candidate also shows significant clinical efficacy against the prevalent UK and South African variants.
 The company has also initiated the development of new constructs to select candidates that can be used as a booster against new strains and plans to initiate clinical trials for these new constructs in the second quarter of 2021.
 
-The primary endpoint of the trial was the occurrence or absence of PCR-confirmed, symptomatic mild, moderate or severe COVID-19 from 7 days after the second dose onward.
-Side effects were monitored in 2,310 participants and were generally considered mild, with low incidences of headache, muscle pain, and fatigue.
+<!-- Which trial does this separate paragraph refer to? Delete? -->
```

Comment:
This is Novavax, here is the original PR: #1063

Reply:
I updated my comment in 65291c7. It's not trivial to keep these sentences about the phase III results because the previous paragraph says they aren't ready yet.  This pull request is huge, so if you're ready, let's merge it and stop making changes.

Reply:
That sounds good to me! I had updated the Novavax text this week but I'm worried it got lost in a merge conflict (resolving them amidst no-childcare-chaos is super difficult, it turns out.) It didn't take long to update, though, so I can fix it again if I can't figure out what happened!